### ISSUE TITLE
[BUG] Incorrect # of total dependencies

### ISSUE THREAD

**Issue by SriKrishnaPaparaju** [fabric8-analytics/fabric8-analytics-vscode-extension#322]
**Describe the bug**
LSP calls show an alert of 132 dependencies with NO CVEs found. The stack report show (a) a total of 9 CVEs (b) total dependencies as 289 . The # of analyzed and unknown dependencies (112,89) do not add up to total # (289)
**To Reproduce**
Try stack report https://gist.github.com/SriKrishnaPaparaju/58535d622dd62da1e2afe4b7c2b380e5

**Expected behavior**

**Screenshots**
<img width="1495" alt="Screen Shot 2019-08-07 at 7 07 31 AM" src="https://user-images.githubusercontent.com/42730908/62588076-0fe07a00-b8e2-11e9-9188-f47edddb9b49.png">

**Logs**
If applicable, attach logs.

**VSCode(please complete the following information):**
 - OS: [e.g. macOS]
 - VSCode version [e.g. 1.33.1]
 - Dependency Analytics Version [e.g. 0.0.22]

**CHE7/Theia(please complete the following information):**
 - OS: [e.g. macOS]
 - Browser [e.g. chrome]
 - CHE/Theia version 
 - Dependency Analytics Version [e.g. 0.0.22]

**Additional context**

**Comment by arajkumar** [comment]
Here is the stack analysis API response,

https://gist.github.com/arajkumar/c59acddb10ba71b7d4b153c879b2b9a5

**Comment by SriKrishnaPaparaju** [comment]
are you able to reproduce this problem?

**Comment by arajkumar** [comment]
<img width="625" alt="Screen Shot 2019-08-09 at 8 24 58 PM" src="https://user-images.githubusercontent.com/3874763/62789696-23106700-bae7-11e9-8e59-7c9f0c12856d.png">

From the response data I could see that `Total Dependencies` is based on `dependencies` JSON field which is `289`,  `Analysed Dependencies` is based on `analysed_dependecies` which is `112`, `Unknown Dependencies` is based on `unknown_dependencies` which is `89`.

I see the duplicates in both `dependencies` and `analysed_dependencies` fields. When I convert those arrays to sets, then counts are match,

<img width="624" alt="Screen Shot 2019-08-09 at 8 40 31 PM" src="https://user-images.githubusercontent.com/3874763/62789686-1d1a8600-bae7-11e9-9d42-32a91704554f.png">

@krishnapaparaju I think the duplicates should be handled from the API side. What's your thought?

**Comment by SriKrishnaPaparaju** [comment]
Yes, this needs to be handled at platform level

**Comment by arajkumar** [comment]
>>LSP calls show an alert of 132 dependencies with NO CVEs found. 

It is a know issue with property based dependency version, refer #258.

**Comment by miteshvp** [comment]
@SriKrishnaPaparaju @arajkumar - As discussed, we cannot remove those duplicates, else we loose the ability to tag a direct dependency affected by any of the transitive dependencies. 
Let me know what else needs to be done on my side.

**Comment by SriKrishnaPaparaju** [comment]
@miteshvp Ack. @arajkumar can we fix on the UI side ?

**Comment by miteshvp** [comment]
Unassigning myself. @arajkumar - you may want to assign this to yourself?

**Comment by arajkumar** [comment]
@SriKrishnaPaparaju , we no longer show `Total Dependency` count in the report after #330 . Do you think it is still relevant as a bug?

You can download VSIX pre-release from [release page](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/releases/download/2019.8.23/fabric8-analytics-2019.8.23.vsix) to access latest changes.

**Comment by arajkumar** [comment]
No longer relevant with new card, hence closing it.
